rpi3.mk

```diff
@@ -31,13 +32,21 @@ RPI3_BOOT_CONFIG	?= $(RPI3_FIRMWARE_PATH)/config.txt
 RPI3_UBOOT_ENV		?= $(ROOT)/out/uboot.env
 RPI3_UBOOT_ENV_TXT	?= $(RPI3_FIRMWARE_PATH)/uboot.env.txt
 RPI3_STOCK_FW_PATH	?= $(ROOT)/firmware
-RPI3_STOCK_FW_PATH_BOOT	?= $(RPI3_STOCK_FW_PATH)/boot
+ifeq ($(PLATFORM),rpi3-b)
+RPI3_STOCK_FW_PATH_BOOT ?= $(RPI3_STOCK_FW_PATH)/boot
```

Comment:
When first looking at this I ask myself why does this have to be on different paths (line 36 and 38, besides the PLATFORM)? I guess you didn't want to deal with the "firmware" git? In that case I believe it's better to actually [create another xml/manifest file](https://optee.readthedocs.io/building/gits/manifest.html) for Rpi3-b+.